public/static/partials/patch_trigger_alias_modal.html

```diff
@@ -0,0 +1,49 @@
+<md-dialog style="width:650px">
```

Comment:
Is the work to make patch triggers do something going to be done in another ticket? In which case this should be hidden for now, so that current project admins don't see it and expect the feature to be complete.